### ISSUE TITLE
final verbose should not show absolute path

### ISSUE THREAD

**Issue by jcohenadad** [neuropoly/spinalcordtoolbox#1164]
~~~
fslview /Users/julien/temp/Subject_1_t2/t2/t2.nii.gz t2_seg.nii.gz -l Red -b 0,1 -t 0.7 &
~~~

replace with (path from where function it's called):
~~~
fslview t2.nii.gz t2_seg.nii.gz -l Red -b 0,1 -t 0.7 &
~~~

**Comment by benjamindeleener** [comment]
The input data may not be at the same place as the output, that is why I've put the full absolute path.

**Comment by jcohenadad** [comment]
the idea would be to use what is passed in "-i", and since the current folder hasn't changed the proper path will be called.

regarding the output: it is currently written as "fname_seg" (without the absolute path) so it should be safe

**Comment by benjamindeleener** [comment]
Fixed in commit f30dc459078be9fb8ddf14a684b3d5c5d1de6e03
